src/chunk.c

```diff
@@ -1919,10 +2023,10 @@ chunk_resurrect(const Hypertable *ht, int chunk_id)
  * context. The returned chunk needs to be copied into another memory context in
  * case it needs to live beyond the lifetime of the other data.
  */
-static int
-chunk_point_find_chunk_id(const Hypertable *ht, const Point *p)
+int32
+ts_chunk_point_find_chunk_id(const Hypertable *ht, const Point *p, bool lock_slices)
```

Comment:
Probably good to extend the comment for the function with what the `lock_slices` parameter does.